compiler/src/iree/compiler/DispatchCreation/SplitReduction.cpp

```diff
@@ -24,8 +26,24 @@ namespace mlir::iree_compiler::DispatchCreation {
 
 // TODO(thomasraoux): Move to attributes.
 static llvm::cl::opt<int64_t>
-    splitReductionRatio("iree-dispatch-creation-split-matmul-reduction",
-                        llvm::cl::desc("split ratio"), llvm::cl::init(1));
+    splitMatmulReductionRatio("iree-dispatch-creation-split-matmul-reduction",
+                              llvm::cl::desc("split ratio"), llvm::cl::init(1));
+
+static llvm::cl::opt<bool> enableStaticArgmaxSplit(
```

Comment:
I don't think we want these options if we are trying to turn it on by default. You need to make it two values

1. The minimum size of the reduction dimension that triggers the split
2. The size of (one of the) the split dimension, I think the size of the reduction dimension in the first operation.

You can set the value of 1 to default to 128*1024 and the second to be 128.

Reply:
I see the reason of having two flags now, it makes sense to me. Thanks!